llvm/test/CodeGen/AIE/aie2/aie2bundle.mir

```diff
@@ -16,7 +16,7 @@ body:             |
     ; CHECK: BUNDLE implicit-def $r0, implicit $r0, implicit $r1 {
     ; CHECK-NEXT:   $r0 = OR $r0, $r1
     ; CHECK-NEXT: }
-    ; CHECK-NEXT: BUNDLE implicit-def $r3, implicit $r1, implicit $r2 {
```

Comment:
This looks like a functional change. I'm not aware that we need bundle operands, but I don't understand what changed.

Reply:
The implicit operands are added when calling `finalizeBundles`. In the finalize pass, we only call it on standalone instructions. For already-bundled instructions, the BUNDLE is left as is.